CMakeLists.txt

```diff
@@ -1,9 +1,15 @@
-cmake_minimum_required(VERSION 3.16)
+cmake_minimum_required(VERSION 3.10)
 project(SigScan)
 
 set(CMAKE_CXX_STANDARD 17)
+set(Boost_USE_STATIC_LIBS ON)
```

Comment:
Has to be before `find_package()`

Reply:
before:
* 291K    SigScan

after:
* 605K    SigScan
* 2.9M    SigScan_static